guides/common/modules/snip_supported-client-operating-systems.adoc

```diff
@@ -0,0 +1,27 @@
+ifndef::orcharhino,satellite[]
+* Amazon Linux
+* Debian
+endif::[]
+ifndef::orcharhino[]
+* {EL} 9 and 8
+endif::[]
+ifdef::satellite[]
+* {EL} 7 and 6 with the https://www.redhat.com/en/resources/els-datasheet[ELS Add-On]
+endif::[]
+ifndef::orcharhino,satellite[]
+* Fedora
+* OpenSUSE
+* {SLES}
+* Ubuntu
+endif::[]
+ifdef::orcharhino[]
+* AlmaLinux
+* Amazon Linux
+* CentOS
+* Debian
+* Oracle Linux
+* Red Hat Enterprise Linux
```

Comment:
We cannot use this attribute because in downstream, we use "RHEL" as a placeholder "Client OS" until I have fixed all instances of "RHEL/EL/any client OS".